plugin/trino-delta-lake/src/test/resources/deltalake/v2_checkpoint_parquet/README.md

```diff
@@ -0,0 +1,13 @@
+Data generated using Delta Lake 3.0.0:
+
+```sql
+SET spark.databricks.delta.checkpointV2.topLevelFileFormat = parquet;
```

Comment:
```suggestion
SET spark.checkpointV2.topLevelFileFormat = parquet;
```

This seems incorrect

https://github.com/delta-io/delta/blob/7d41fb7bbf63af33ad228007dd6ba3800b4efe81/spark/src/main/scala/org/apache/spark/sql/delta/sources/DeltaSQLConf.scala#L743-L754

Reply:
Good catch, thanks!

Reply:
I confirmed `spark.checkpointV2.topLevelFileFormat` doesn't work. The previous `spark.databricks.delta.checkpointV2.topLevelFileFormat` is the right session name. 

https://github.com/delta-io/delta/blob/7d41fb7bbf63af33ad228007dd6ba3800b4efe81/spark/src/main/scala/org/apache/spark/sql/delta/sources/DeltaSQLConf.scala#L33-L37